projects/can_dump/scripts/dump_system.py

```diff
@@ -157,7 +157,7 @@ def main():
     parser.add_argument('device', help='Serial device or "slcan0"')
     args = parser.parse_args()
 
-    log_file = '{}/system_can_{}.log'.format(args.log_dir, datetime.datetime.now())
+    log_file = '{}/system_can_{}.log'.format(args.log_dir, datetime.datetime.now().strftime('%Y-%m-%d %H.%M.%S.%f'))
```

Comment:
The build failed because this line is too long. Should pass if you split the line